pkgs/development/ruby-modules/gem-config/default.nix

```diff
@@ -179,6 +179,10 @@ in
     buildInputs = [ openssl ];
   };
 
+  exif = attrs: {
+    buildFlags = [ "--with-exif-dir=${libexif}" ];
```

Comment:
Does this require libexif in buildInputs?

Reply:
You are absolutely correct. I wasn't too familiar with nixpkgs at a time of pull request creation. It is fixed now, many thanks.